tests/recipes/cad1/task1/test_enhance.py

```diff
@@ -0,0 +1,162 @@
+"""Tests for the enhance module"""
+# pylint: disable=import-error
+
+import numpy as np
+import torch
+from torchaudio.pipelines import HDEMUCS_HIGH_MUSDB
+
+from clarity.enhancer.compressor import Compressor
+from clarity.enhancer.nalr import NALR
+from recipes.cad1.task1.baseline.enhance import (
+    apply_baseline_ha,
+    decompose_signal,
+    get_device,
+    map_to_dict,
+    process_stems_for_listener,
+    separate_sources,
+)
+
+
+def test_map_to_dict():
+    """Test that the map_to_dict returns the expected mapping"""
+    sources = np.array([[1, 2], [3, 4], [5, 6]])
+    sources_list = ["a", "b", "c"]
+    output = map_to_dict(sources, sources_list)
+    expected_output = {
+        "left_a": 1,
+        "right_a": 2,
+        "left_b": 3,
+        "right_b": 4,
+        "left_c": 5,
+        "right_c": 6,
+    }
+    assert output == expected_output
+
+
+def test_decompose_signal():
+    """Takes a signal and decomposes it into VDBO sources using the HDEMUCS model"""
+    np.random.seed(123456789)
+    # Load Separation Model
+    model = HDEMUCS_HIGH_MUSDB.get_model().double()
+    device = torch.device("cpu")
+    model.to(device)
+
+    # Create a mock signal to decompose
+    sample_rate = 8000
+    duration = 1
+    signal = np.random.uniform(size=(1, 2, sample_rate * duration))
+    # Call the decompose_signal function and check that the output has the expected keys
+    output = decompose_signal(model, signal, sample_rate, device)
+
+    expected_results = np.load(
+        "../../../resources/test_enhance.test_decompose_signal.npy", allow_pickle=True
```

Comment:
With `RESOURCES` defined we can instead use the following here and in other places.

```suggestion
        RESOURCES / "test_enhance.test_decompose_signal.npy", allow_pickle=True
```